src/routers/employer.ts

```diff
@@ -2,17 +2,16 @@ import express from 'express'
 import tokenVerify from '../middlewares/tokenVerify'
 
 import {
-  localLogin,
   registerEmployer,
   createJobPost,
   updateJobPost,
   deleteJobPostbyId,
   updateEmployer,
 } from '../controllers/employer'
-
+import { jobSeekerLocalLogin } from '../controllers/login'
 const router = express.Router()
 
-router.post('/login/local', localLogin)
+router.post('/login/local', jobSeekerLocalLogin)
```

Comment:
You shouldn't import and use the new login controller in both `employer` and `jobseeker` router. Make a separate router for the new controller also